shared/common-adapters/copy-text.js

```diff
@@ -81,9 +81,11 @@ class _CopyText extends React.Component<Props, State> {
             Reveal
           </Text>
         )}
-        <Button type="Primary" style={styles.button} onClick={this.copy}>
-          <Icon type="iconfont-clipboard" color={globalColors.white} fontSize={isMobile ? 20 : 16} />
-        </Button>
+        <Box2 direction="horizontal" style={styles.buttonContainer}>
```

Comment:
The issue I found that this bug was coming from was the fact that the button was being absolutely positioned and assigned a constant height. So I put the button back in the flex row and gave it some padding so that it had the desired look. The problem with that, however, is that you can't `justifySelf: 'flex-end'` in flexbox so I wrapped it in this flexbox with `flexGrow: 1` and `justifyContent: `flex-end` to achieve the same effect. This box is necessary unless you have a different suggestion on how to position the button.

I suppose an alternative would be to `position: absolute` the button, give it a height of 100% and the same top/bottom padding as its parent but that leads to some style duplication and I think leaving things in the normal content flow is more flexible when possible.

Reply:
sorry github styling hid a very important word above, edited

Reply:
oh this probably could use `<ButtonBar>`, trying it out now

Reply:
@chrisnojima Works fine with `ButtonBar`, but I had to overwrite some of `ButtonBar`'s default styles so I couldn't get rid of `styles.buttonContainer`